src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const persons = document.querySelectorAll('ul li');
+
+const sorted = [...persons].sort((a, b) => {
+  return Number(b.dataset.salary
+    .slice(1)
+    .split(',')
+    .join(''))
+  - Number(a.dataset.salary
+    .slice(1)
+    .split(',')
+    .join(''));
+});
+
+list.innerHTML = '';
+
+for (const li of sorted) {
+  list.appendChild(li);
+};
```

Comment:
after `sort` method you can use `forEach` method and no need in this last part of the code.
E.g.
`[...persons].sort((a,b) => ...).forEach(person => list.append(person))`